docs/book/getting-started/zenml-pro/hybrid-deployment-helm.md

```diff
@@ -0,0 +1,670 @@
+
+### 1. Create Kubernetes Resources for Workload Manager
+
+Create a dedicated namespace and service account:
+
+```bash
+kubectl create namespace zenml-workload-manager
+kubectl -n zenml-workload-manager create serviceaccount zenml-runner
+```
+
+### 2. Configure Workload Manager in Helm Values
+
+Add environment variables to your `zenml-hybrid-values.yaml`:
+
+**Option A: Kubernetes-based (Simplest)**
+
+```yaml
+zenml:
+  environment:
+    ZENML_SERVER_WORKLOAD_MANAGER_IMPLEMENTATION_SOURCE: zenml_cloud_plugins.kubernetes_workload_manager.KubernetesWorkloadManager
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_NAMESPACE: zenml-workload-manager
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_SERVICE_ACCOUNT: zenml-runner
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_RUNNER_IMAGE: 715803424590.dkr.ecr.eu-central-1.amazonaws.com/zenml-pro-server:<ZENML_OSS_VERSION>
+```
+
+**Option B: AWS-based (if running on EKS)**
+
+```yaml
+zenml:
+  environment:
+    ZENML_SERVER_WORKLOAD_MANAGER_IMPLEMENTATION_SOURCE: zenml_cloud_plugins.aws_kubernetes_workload_manager.AWSKubernetesWorkloadManager
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_NAMESPACE: zenml-workload-manager
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_SERVICE_ACCOUNT: zenml-runner
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_BUILD_RUNNER_IMAGE: "true"
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_DOCKER_REGISTRY: <your-ecr-registry>
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_ENABLE_EXTERNAL_LOGS: "true"
+    ZENML_AWS_KUBERNETES_WORKLOAD_MANAGER_BUCKET: s3://your-bucket/zenml-logs
+    ZENML_AWS_KUBERNETES_WORKLOAD_MANAGER_REGION: us-east-1
+```
+
+**Option C: GCP-based (if running on GKE)**
+
+```yaml
+zenml:
+  environment:
+    ZENML_SERVER_WORKLOAD_MANAGER_IMPLEMENTATION_SOURCE: zenml_cloud_plugins.gcp_kubernetes_workload_manager.GCPKubernetesWorkloadManager
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_NAMESPACE: zenml-workload-manager
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_SERVICE_ACCOUNT: zenml-runner
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_BUILD_RUNNER_IMAGE: "true"
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_DOCKER_REGISTRY: <your-gcr-registry>
+```
+
+### 3. Configure Pod Resources (Optional but Recommended)
+
+```yaml
+zenml:
+  environment:
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_POD_RESOURCES: '{"requests": {"cpu": "500m", "memory": "512Mi"}, "limits": {"cpu": "2000m", "memory": "2Gi"}}'
+    ZENML_KUBERNETES_WORKLOAD_MANAGER_TTL_SECONDS_AFTER_FINISHED: 86400
+    ZENML_SERVER_MAX_CONCURRENT_TEMPLATE_RUNS: 5
+```
+
+### 4. Redeploy with Updated Values
+
+```bash
+helm upgrade zenml zenml/zenml \
+  --namespace zenml-hybrid \
+  --values zenml-hybrid-values.yaml
+```
+
+## Step 8: Configure Environment Variables (Advanced)
+
+For advanced configurations, you can set additional environment variables in your Helm values:
+
+```yaml
+zenml:
+  environment:
+    ZENML_LOGGING_LEVEL: INFO
+    ZENML_ANALYTICS_OPT_IN: "false"
+    # Add other environment variables as needed
+```
+
+## Database Configuration Examples
```

Comment:
This entire section is unnecessary. We already have a complete section on how to configure the helm chart for OSS server deployments. Duplicating oversimplified parts of that here - and incorrectly at that - isn't going to help anyone. Better to link to the correct and fully detailed OSS helm documentation here instead.